.travis.yml

```diff
@@ -1,13 +1,21 @@
 language: go
+go_import_path: github.com/davecgh/go-spew
```

Comment:
Is this line really needed? As far as I know, this project doesn't use a vanity import path, so it shouldn't be needed.

Reply:
Without this line, I can't get Travis to successfully run the tests by pushing to github.com/kevinburke/go-spew. It clones into kevinburke/go-spew and imports fail.